src/cobra/core/reaction.py

```diff
@@ -882,6 +882,12 @@ def delete(self, remove_orphans: bool = False) -> None:
         )
         self.remove_from_model(remove_orphans=remove_orphans)
 
+    def __getstate__(self) -> dict:
```

Comment:
`dict` -> `Dict` for Python <= 3.9. A "Returns" section in the docstring would be perfect.

Reply:
Done